### DISCUSSION TITLE
"Resolving API version 0.0.0" Warning is showing

### DISCUSSION THREAD

**Discussion by beno100** [scripthookvdotnet/scripthookvdotnet#1290] [Q&A]
https://kepkuldes.com/image/D6LSSg 

everything is up to date, but this is anytime showing. 
 
 
 please help thx

**Comment by kagikn** [comment]
Nope, this is the expected behavior as all of the scripts showed in the image are built against a version-less ScriptHookVDotNet that is older than v2.10.0 (which means they want to reference to `ScriptHookVDotNet`, not `ScriptHookVDotNet2`. We redirect it though.) If you don't have any other issues on the scripts, you can continue to use for now.
